packages/ui/src/components/DateInput/__stories__/Range.stories.tsx

```diff
@@ -1,25 +1,69 @@
 import type { StoryFn } from '@storybook/react'
+import type { ComponentProps } from 'react'
 import { useState } from 'react'
 import { DateInput } from '..'
+import { Stack } from '../../Stack'
 
-export const Range: StoryFn = args => {
+const months = [
```

Comment:
```suggestion
const MONTHS = [
```

Usually we set constant like this in uppercase to differentiate them from dynamic variables